src/main/java/org/broadinstitute/hellbender/tools/spark/sv/ContigsCollection.java

```diff
@@ -85,50 +49,52 @@ public String toPackedFasta(){
      * @param packedFastaLine
      */
     @VisibleForTesting
-    protected static ContigsCollection fromPackedFasta(final String packedFastaLine) {
+    protected static ContigsCollection fromPackedFasta(final long assemblyID, final String packedFastaLine) {
```

Comment:
About removing `ContigsCollection`, another related concern is that when we have fermi-lite assembly as the assembly tool, a similar collection concept might be necessary. But currently the jni binding was simply returning a list of MAG format vertices.
When we request GFA format (a graph) from fermi-lite, the returned struct will be a whole atomic object instead of a container of something.
So it is not a concern there.